### PR TITLE
Feature/at 7294 phone input tech debt

### PR DESCRIPTION
**Pull request by DGabriel-CCPO**

### CODE REVIEW

File: src/components/ATATPhoneInput.vue [hunk 1]
```diff
@@ -232,18 +274,30 @@ export default class ATATPhoneInput extends Vue {
...
+        regex:"^[0-9]{1,6}?",
+        placeholder: "",
+        jitMasking: true
+      }).mask(extensionTextField);
+    });
+  }
   private mounted(): void {
     this.searchResults = [...this.countries];
     this.setPhoneMask();
+    this.setExtensionMask();
```
Comment (tonysumner-ccpo) [inline src/components/ATATPhoneInput.vue:300]: - [ ] more likely not to give stooopid & usually harmless console errors if the `setExtensionMask` is only called if there is a phone extension.
`if (phoneExtension) { this.setExtensionMask();}`

File: src/steps/01-AcquisitionPackageDetails/COR_ACOR/ContactInfoForm.vue [hunk 2]
```diff
@@ -88,6 +88,15 @@
...
 
+        <ATATPhoneInput
+          id="PhoneNumber"
+          label="Phone number"
+          class=" width-100"
+          :value.sync="_phone"
+          :country.sync="_selectedPhoneCountry"
+          :extension.sync="_phoneExt"
+        />
+
```
Comment (tonysumner-ccpo) [inline src/steps/01-AcquisitionPackageDetails/COR_ACOR/ContactInfoForm.vue:99]: - [ ] the prop `  @Prop({ default: true }) private phoneExtension!: boolean;` in the ATATPhoneInput.vue is not being used here. Is it necessary?

File: src/steps/01-AcquisitionPackageDetails/ContactInfo.vue [hunk 3]
```diff
@@ -106,6 +106,19 @@
...
+          label="Your phone number"
+          id="ContactPhone"
+          :class="{ 'mb-10': selectedRole === 'CIVILIAN' }"
+          :value.sync="selectedPhoneNumber"
+          :country.sync="selectedPhoneCountry"
+          :extension.sync="phoneExtension"
+          :rules="[
+            $validators.isPhoneNumberValid(
+              this.selectedPhoneCountry
+            ),
```
Comment (tonysumner-ccpo) [inline src/steps/01-AcquisitionPackageDetails/ContactInfo.vue:119]: - [ ] the prop `  @Prop({ default: true }) private phoneExtension!: boolean;` in the ATATPhoneInput.vue is not being used here. Is it necessary?

**Comment by tonysumner-ccpo** [comment]
- [ ] Phone number validation stretches under the extension and shouldn't.  Should only be below the phone number control. Have sent you the code in the slack.  
 
image

**Review by ericyoungquist-ccpo — APPROVED** [review]

**Pull request merged**